app/javascript/dashboard/components/widgets/conversation/conversationBulkActions/LabelActions.vue

```diff
@@ -0,0 +1,265 @@
+<template>
+  <div v-on-clickaway="onClose" class="labels-container">
+    <div class="triangle">
+      <svg height="12" viewBox="0 0 24 12" width="24">
+        <path
+          d="M20 12l-8-8-12 12"
+          fill="var(--white)"
+          fill-rule="evenodd"
+          stroke="var(--s-50)"
+          stroke-width="1px"
```

Comment:
This works, but it feels like repetition. If it is a small work, can we refactor this to a popover component.